src/manage.c

```diff
@@ -462,9 +451,6 @@ static msg_t manage_track_thread(void *arg)
 
 void manage_track_setup()
```

Comment:
Splitting to manage_acq.c and manage_track.c would make the code more readable IMO